doc/library/tensor/random/basic.rst

```diff
@@ -43,3 +43,11 @@ Reference
     :class:`Op` that draws random numbers from a :class:`numpy.random.RandomState` object.
     This :class:`Op` is parameterized to draw numbers from many possible
     distributions.
+
+Distributions
+==============
+
+Aesara can produce :class:`RandomVariable`\s that draw samples from many different statistical distributions, using the following :class:`Op`\s.
+
+.. autoclass:: aesara.tensor.random.basic.GammaRV
+   :members: __call__
```

Comment:
This is a good addition.  It reminds me that we still need to do this (i.e. replace manually entered method documentation with `autoclass` directives) for _all_ references to objects/classes in the documentation.

Reply:
Yes. I looked at the documentation of the `aesara.tensor` module and was surprised to see it was all manually entered.

We'll also need to document the `RandomStream` approach better, and we can pull the corresponding Ops's docstrings.

Reply:
We should also provide an `aesara.tensor.random.default_rng` interface to `RandomStream`.